src/aura/STG_CMP_System/STG_CMP_SystemHelper.js

```diff
@@ -1,12 +1,7 @@
 ({
-	onSelectChange : function(component) {
-		var selectedVal = component.find("accRecTypes").get("v.value");
-		component.set("v.accRecTypeId", selectedVal);
-	},
-
 	saveAccRecType : function(component) {
 		//Get selected value
-		var accRecTypesDropDown = component.find("accRecTypes");
+		/*var accRecTypesDropDown = component.find("accRecTypes");
```

Comment:
@ceiroa Do we still need all of this commented code? The function is still called but everything is commented out. Looks like functionality was replaced by the new recordtype dropdown cmp.